pages/base.py

```diff
@@ -0,0 +1,103 @@
+import settings
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import StaleElementReferenceException, TimeoutException
+
+
+class BaseElement(object):
+    default_timeout = settings.TIMEOUT
+
+    def __init__(self, driver):
+        self.driver = driver
```

Comment:
according to the way we launch driver in travis, this should call that function to setup a driver instead of using driver directly to keep it consistent. And also allow for remote testing.

Reply:
Because we make multiple pages and elements within a page, setting up the driver within the page object yields multiple browsers. So instead I opted to just set up the driver in the test. Is there a way to avoid this?

Reply:
I think the driver should be initialized by the test even if there is a way to avoid this. Suppose there comes a time when we want to test collaborative editing for OSF, it would need two browsers to do so. We can then pass two different browser to two page object instances, which is something we cannot do if we choose to initialize the driver within the page object.